flagsmith-core.ts

```diff
@@ -249,7 +269,7 @@ const Flagsmith = class {
     oldFlags:IFlags|null= null
     onChange:IInitConfig['onChange']|null= null
     onError:IInitConfig['onError']|null = null
-    identity?: string|null= null
+    identity?: IIdentity|null= null
     ticks: number|null= null
     timer: number|null= null
     traits:ITraits|null= null
```

Comment:
I think this needs to change too, how do I initialise with transient traits?

Reply:
The `ITraits` type has been extended.

Reply:
In addition to this, will this persist transient traits in AsyncStorage and load them in onRetrievedStorage successfully?

Reply:
No idea tbh!

Reply:
I think it will, but we'll find out in the tests